src/patternfly/patternfly-charts.scss

```diff
@@ -256,6 +256,9 @@
   // Scatter Chart
   --pf-chart-stack--data--stroke--Width: #{$pf-chart-stack--data--stroke--Width};
 
+  // Threshold strokeDasharray: '6,6'
```

Comment:
nit - can you remove `'6,6'`? Also if we expect we'll add other "Threshold" vars at some point, I think this comment can just say `// Threshold`

Reply:
sure, no problem